### ISSUE TITLE
Could not find a declaration file for module '@vis.gl/react-google-maps'.  v0.2.0

### ISSUE THREAD

**Issue by MauricioRobayo** [visgl/react-google-maps#61]
Hello!

I'm testing the package and installed the latest version at the moment ([v0.2.0](https://github.com/visgl/react-google-maps/releases/tag/v0.2.0)).

After installing the package and creating a component based on [this example](https://github.com/leighhalliday/next-google-maps/blob/main/src/app/intro/page.tsx), I get the following typescript error:

```
Could not find a declaration file for module '@vis.gl/react-google-maps'. 
```

Downgrading to v0.1.2 seems to fix the issue.

Here is a test repo: https://github.com/MauricioRobayo/google-maps-test/blob/main/src/app/GoogleMap.tsx

Steps to reproduce:

1. Clone the repo: https://github.com/MauricioRobayo/google-maps-test
2. Install dependencies: `npm install`
3. Build the app: `npm run build`

The build fails because of the error:

<img width="659" alt="image" src="https://github.com/visgl/react-google-maps/assets/2121481/162b6e84-5265-47d1-9358-19db14f1fd90">

**Comment by usefulthink** [comment]
I think I've seen that one before, let me see if it's an easy fix.

**Comment by usefulthink** [comment]
I just published version 0.2.1, which should have a fix for this. Can you verify?

**Comment by MauricioRobayo** [comment]
Verified, it is fixed. Thanks @usefulthink
